src/main/java/org/apache/sysml/hops/HopsException.java

```diff
@@ -45,4 +45,14 @@ public HopsException(String message, Throwable cause) {
         super(message, cause);
     }
 
+    /** If the condition fails, print the message formatted with objects. */
```

Comment:
If you add javadoc, please specify the parameters and conditions, as otherwise this will throw javadoc warnings/errors.